gradle/posix.gradle

```diff
@@ -104,3 +104,21 @@ kotlin {
 }
 
 project.ext.set("hasNative", true)
+
+// hack for 1.4-M2, remove after
+// see https://youtrack.jetbrains.com/issue/KT-39302
+afterEvaluate {
+    def metadataTarget = kotlin.targets.getByName("metadata")
+
+    def currentHostKonanTarget = hostManager.Companion.getHost()
+
+    if (metadataTarget != null) {
```

Comment:
Unlike `findByName`, `getByName` never returns null, it throws an exception if the element is absent. However, in Kotlin MPP modules that have `kotlin.targets`, there's always a target under the name `metadata`. So it seems that the null check is redundant.